aggregate.R

```diff
@@ -3,56 +3,78 @@
 'aggregate
 
 Usage:
-  aggregate.R <sqlite_file> -o <file>
+  aggregate.R -s <sqlite_file> -o <file> [-t <sc_type>]
```

Comment:
after reviewing, I don't think this change is ready for prime time. A couple more conditional statements that only behave when `sc_type` is specified are required.